example/viewModeChangeExample.js

```diff
@@ -0,0 +1,34 @@
+var DateTime = require('../DateTime.js');
+var React = require('react');
+var ReactDOM = require('react-dom');
+var moment = require('moment');
+
+var Wrapper = React.createClass({
+	getInitialState: function() {
+		return {
+			viewMode: 'time'
+		};
+	},
+
+	updateView: function(format) {
+		console.log('changing state');
```

Comment:
Can this be `console.log('changing viewMode to days');`? It's nice to see what the `viewMode` will change to.